src/components/common/RecipientsTable/RecipientTable.js

```diff
@@ -334,16 +337,21 @@ const RecipientTable = ({
 
   return (
     <div style={{}}>
-      <EditRecipientForm
-        visible={editing}
-        onCreate={() => {
-          alert('This is submited');
-        }}
-        onCancel={() => {
-          setEditing(false);
-        }}
-        recipient_id={key}
-      />
+      {editing ? (
```

Comment:
This is a a great way of getting the edit form modal to populate with existing values from the database. Using the value of "editing" as a conditional to pass down the recipientToEdit is a clever solution.

Reply:
This was a great implementation to get the component to rerender to update those values!

Reply:
Jaden came up with that solution, it was indeed clever!